src/utils/Api.js

```diff
@@ -0,0 +1,126 @@
+class Api {
+  constructor({ baseURL, headers }) {
+    this._baseURL = baseURL;
+    this._headers = headers;
+  }
+
+  getCurrentUser() {
+    return fetch(`${this._baseURL}/users/me`, {
+      method: "GET",
+      headers: this._headers,
+    })
+      .then((res) => {
+        return this._checkResponse(res);
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
You should **not** use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request, and this end is in `index.js`   after all `then` blocks. And in real projects the `catch` blocks usually open error popups and so on.